.coveragerc

```diff
@@ -575,6 +575,7 @@ omit =
     homeassistant/components/solaredge/sensor.py
     homeassistant/components/solaredge_local/sensor.py
     homeassistant/components/solax/sensor.py
+    homeassistant/components/soma/*
```

Comment:
The config flow requires 100% test coverage, so don't exclude that module.

Reply:
Does this mean I have to write some automated tests? Or should I just exclude specific files from the Soma folder?

Reply:
You need to write tests for the config flow module. The other modules can be excluded here.

Reply:
Oh God. So how many of these tests are actually required? Who will be running this test suite for the new versions? For most tests hardware would also be needed. I can send hardware to whoever needs them for testing. 
Is it enough if I write tests for successful connection, wrong ip/port, cancelled setup? Or would more be needed.
I'm not sure if I have to support more than one hub at the moment. The guide said keep it simple for the first version. I don't think many people would use more than one hub.